tests/tck/features/match/SeekById.feature

```diff
@@ -317,6 +317,28 @@ Feature: Match seek by id
       """
     Then the result should be, in any order:
       | v |
+    When executing query:
```

Comment:
Don't put them in this file. It's not vid seek.

Reply:
done

Reply:
But I don't see the modification..